File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.java
    class Solution {
        public int solution(int[][] sizes) {
            int answer = 0;
            int biggerSideMax = 0;
            int smallerSideMax = 0;

            for (int[] size : sizes) {
                int currentBiggerSide = Math.max(size[0], size[1]);
                int currentSmallerSide = Math.min(size[0], size[1]);

                if (biggerSideMax < currentBiggerSide) biggerSideMax = currentBiggerSide;
                if (smallerSideMax < currentSmallerSide) smallerSideMax = currentSmallerSide;
            }

            return biggerSideMax * smallerSideMax;
        }
    }

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    가로 = []
    세로 = []

    for i in sizes:
        if i[0] < i[1]:
            가로.append(i[0])
            세로.append(i[1])
        else:
            가로.append(i[1])
            세로.append(i[0])

    가로.sort()
    세로.sort()
    sum = 가로[len(가로) - 1] * 세로[len(세로) - 1]
    return sum